src/plugins/crypto.c

```diff
@@ -53,6 +53,12 @@
 #define DEFAULT_LUKS_KEYSIZE_BITS 256
 #define DEFAULT_LUKS_CIPHER "aes-xts-plain64"
 
+#ifdef LIBCRYPTSETUP_23
+#define DEFAULT_INTEGRITY_TAG_SIZE 0
+#else
+#define DEFAULT_INTEGRITY_TAG_SIZE 4
+#endif
```

Comment:
This is worth a comment.

Reply:
I've added a comment explaining the defaults. The `4` for older versions of cryptsetup isn't really the best choice (it will work only with crc32, truncating the tag for "longer" hashes like sha256) but I don't want to add another logic for calculating the correct tag size for hashes, that should be up to the caller. (And this probably wouldn't be used with older cryptsetup anyway, I don't plan to backport this to 2.x).